README.md

```diff
@@ -17,12 +17,12 @@ The PDB format has not been officially documented, presenting a challenge for ot
 toolsets (such as Clang/LLVM) that want to work with Windows or the Visual Studio debugger. We want
 to help the Open Source compilers to get onto the Windows platform.
  
-The majority of content on this repo is presented as actual source files from the VC++ compiler 
+The majority of content on this repository is presented as actual source files from the VC++ compiler 
 toolset. Source code is the ultimate documentation :-) We hope that you will find it helpful. If you 
 find that you need other information to successfully complete your project, please enter an
 [Issue](https://github.com/microsoft/microsoft-pdb/issues) letting us know what information you need.
 
-##Start here
+<h2>Start here</h2>
```

Comment:
This makes no much sense, and sounds less markdown-ish, IMVHO.

It doesn't work though in their current form because of lack of space between ## and text

Reply:
### I have just started with open source and some school seniors suggested to start contributing in any form you can. I thought I could help with the readability of the readme here, though in retrospection should have gone through some more resources about it. Also realized that microsoft is really not a place for a bigenner lol. Any advice for a noobie or feedback is greatly appreciated.
Thanks.